fe/fe-core/src/main/java/com/starrocks/system/Backend.java

```diff
@@ -78,6 +81,9 @@ public enum BackendState {
     @SerializedName(value = "d")
     private volatile ConcurrentHashMap<String, DiskInfo> disksRef;
 
+    @SerializedName("loc")
+    private Map<String, String> location = new HashMap<>();
```

Comment:
The location information would not be persisted by method `write()`?

Reply:
> The location information would not be persisted by method `write()`?

Oh, I know. In the new version, we leverage json to do serialization, so we do not need to manual serialize in method `write()`.